### PR TITLE
fix: type of node on consensus full node steps

### PR DESCRIPTION
**Pull request by rootulp**

### CODE REVIEW

**Review by dabit3 — APPROVED** [review]

**Comment by rootulp** [comment]
I don't have write access so will defer to @dabit3 to merge. Also open to being granted write access.

Screen Shot 2022-07-27 at 7 21 09 PM

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**
